jps-plugin/src/main/kotlin/org/rust/jps/RustBuilder.kt

```diff
@@ -0,0 +1,129 @@
+package org.rust.jps
+
+
+import org.jetbrains.jps.ModuleChunk
+import org.jetbrains.jps.builders.DirtyFilesHolder
+import org.jetbrains.jps.builders.java.JavaSourceRootDescriptor
+import org.jetbrains.jps.incremental.*
+import org.jetbrains.jps.incremental.messages.BuildMessage
+import org.jetbrains.jps.incremental.messages.CompilerMessage
+import org.jetbrains.jps.model.module.JpsModule
+
+import java.io.BufferedReader
+import java.io.File
+import java.io.IOException
+import java.io.InputStreamReader
+import java.util.regex.Pattern
+
+
+class RustBuilder : ModuleLevelBuilder(BuilderCategory.TRANSLATOR) {
+
+
+    @Throws(ProjectBuildException::class, IOException::class)
+    override fun build(context: CompileContext,
+                       chunk: ModuleChunk,
+                       dirtyFilesHolder: DirtyFilesHolder<JavaSourceRootDescriptor, ModuleBuildTarget>,
+                       outputConsumer: ModuleLevelBuilder.OutputConsumer): ModuleLevelBuilder.ExitCode {
+
+        for (module in chunk.modules) {
+            if (module.moduleType !is JpsRustModuleType) {
+                continue
+            }
+
+            context.processMessage(CompilerMessage("cargo", BuildMessage.Kind.INFO, "cargo build"))
+
+            val path = getContentRootPath(module)
+
+            val processBuilder = ProcessBuilder("cargo", "build")
```

Comment:
Will this use `cargo` from the system `PATH`? Should we use `cargo` from `SDK` here?